src/Connection.ts

```diff
@@ -1255,14 +1255,16 @@ export class Connection<
 
 	/**
 	 * Query a predefined object view.
+	 * @param design The namespace of the object view, as defined in io-package.json. Usually the adapter name, e.g. "hm-rpc"
+	 * @param type The type of object.
 	 * @param start The start ID.
 	 * @param end The end ID.
-	 * @param type The type of object.
 	 */
 	getObjectView<T extends ioBroker.ObjectType>(
-		start: string,
-		end: string,
-		type: T,
+		design: 'system' | 'chart' | string,
+		type: T | string,
```

Comment:
Similarly, this also reduces to just `string`.

Reply:
This will reduce to string, thats true... I thought it will keep the info, but it does not. So I'd suggest to remove 'string' and only keep the most common.
Also if we change to string the inheritance of T will not work any more ( I think )

Reply:
I checked again, here it seems to keep the T

Reply:
![grafik](https://user-images.githubusercontent.com/17641229/148074902-032a81e6-8825-4ed0-ba82-b338a4762d36.png)